app/scripts/dimApp.i18n.js

```diff
@@ -636,58 +636,220 @@
+            LockPerk: "Por perk",
+            Missing1: "Faltam itens lendários ou exóticos para construir um set completo!",
+            Missing2: "Faltam itens raros, lendários ou exóticos para construir um set completo!" },
           Loadouts: {
+            Loadouts: "Loadouts",
             Before: "Antes '{{name}}'",
-            Create: "Criar Loadout",
+            Create: "Criar set",
             FromEquipped: "Equipado",
-            Edit: "Editar Loadout",
-            Delete: "Excluir Loadout",
+            Edit: "Editar set",
+            Delete: "Excluir set",
+            ConfirmDelete: "Tem certeza que deseja excluir '{{name}}'?",
             ApplySearch: "Itens = \"{{query}}\"",
-            MaximizeLight: "Maximizar a Luz",
-            ItemLeveling: "Nivelamento de item",
+            MaximizeLight: "Maximizar Luz",
+            ItemLeveling: "Nivelamento de itens",
             GatherEngrams: "Obter engramas",
             GatherEngramsExceptExotics: "Exóticos",
-            FarmingMode: "Modo Farm (mover engramas)",
             RestoreAllItems: "Todos os itens",
-            Loadouts: "Loadouts",
-            AppliedAuto: "Construtor automático de Loadouts",
-            Applied: "Seu loadout de {{amount}} itens foram transferidos para seu {{store}}.",
-            AppliedError: "Nenhum item em seu Loadout pode ser transferido.",
-            AppliedWarn: "Seu Loadout foi parcialmente transferido, mas {{failed}} de {{total}} itens retornaram erros." },
+            LoadoutName: "Nome do set...",
+            Save: "Salvar",
+            SaveAsNew: "Salvar como novo",
+            Cancel: "Cancelar",
+            ItemsWithIcon: "Itens com este ícone serão equipados.",
+            ClickToEquip: "Clicar em um item alterna a equipagem.",
+            AppliedAuto: "Construtor automático de sets",
+            Applied: "Seu set de {{amount}} itens foi transferido para seu {{store}}.",
+            AppliedError: "Nenhum dos itens do seu set pode ser transferido.",
+            AppliedWarn: "Seu set foi parcialmente transferido, mas {{failed}} de {{total}} itens retornaram erros." },
+          Manifest: {
+            Build: "Construindo banco de dados de informações",
+            Download: "Baixando últimas informações da Bungie",
+            Error: "Erro ao carregar informações de Destiny:\n{{error}}\nReinicie para tentar novamente.",
+            Outdated: "Informações desatualizadas",
+            OutdatedExplanation: "A Bungie atualizou seu banco de dados de informações de Destny. Reinicie o DIM para obter as novas informações. Esteja ciente que algumas coisas no DIM podem não funcionar por algumas horas após a Bungie atualizar o Destiny, já que as novas informações ainda estão sendo implementadas em seus servidores.",
+            BungieDown: "A Bungie.net parece estar passando por problemas.",
+            Load: "Carregando informações salvas",
```

Comment:
Done! And BTW, this should be added to https://github.com/DestinyItemManager/DIM/blob/dev/app/scripts/services/dimStoreService.factory.js#L1428